starter_project_backend_nest/src/test/article.spec.ts

```diff
@@ -99,16 +97,16 @@ describe('Article Testing', () => {
 
   describe('DELET Article API', () => {
     test('respose after deleting should be [success]', async () => {
-      const sampleArt = await service.getAllArticle();
+      const sampleArt = await articleService.getAllArticle();
       let sampleId = sampleArt[0]._id;
-      const res = await service.deleteArticleById(sampleId);
-      expect(res).toEqual('success');
+      const deletedArticle = await articleService.deleteArticleById(sampleId);
```

Comment:
after deleting the article you cannot access the article deleted so it's better to check staus code